### ISSUE TITLE
[bilibili] not work on bangumi.bilibil

### ISSUE THREAD

**Issue by yxlharry** [rg3/youtube-dl#11845]
can not work on bangumi.bilibili

www.bilbili.com/video/av  like:
 "http://www.bilibili.com/video/av8193045/" still work

youtube-dl -v http://bangumi.bilibili.com/anime/5802/play#100643
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'http://bangumi.bilibili.com/anime/5802/play#100643']
[debug] Encodings: locale cp936, fs mbcs, out cp936, pref cp936
[debug] youtube-dl version 2017.01.25
[debug] Python version 3.4.4 - Windows-10-10.0.14393
[debug] exe versions: none
[debug] Proxy map: {}
[generic] play#100643: Requesting header
WARNING: Falling back on generic information extractor.
[generic] play#100643: Downloading webpage
[generic] play#100643: Extracting information
ERROR: Unsupported URL: http://bangumi.bilibili.com/anime/5802/play#100643
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmppmszswf3\build\youtube_dl\YoutubeDL.py", line 694, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmppmszswf3\build\youtube_dl\extractor\common.py", line 359, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmppmszswf3\build\youtube_dl\extractor\generic.py", line 2551, in _real_extract
youtube_dl.utils.UnsupportedError: Unsupported URL: http://bangumi.bilibili.com/anime/5802/play#100643

**Comment by yan12125** [comment]
Any globally available video on bangumi.bilibili.com?

**Comment by yxlharry** [comment]
oh..yes you remind me, some ( all?)of bangumi videos may only available in China，for the copyright.

**Comment by yxlharry** [comment]
But I am in China

**Comment by yan12125** [comment]
The label "geo-restricted" indicates that a developer needs a proxy to fix this problem. I'm not in China, and IIRC no major developers are in China. To make things worse, unblock-youku, one of the most famous China proxy, does not work (https://github.com/uku/Unblock-Youku/issues/719), so this issue may be postponed. I was asking for a globally available video so that solving this becomes easier.

**Comment by yxlharry** [comment]
Thanks! Understood, I'll use proxy and try to find some globally available video tomorrow.

**Comment by yxlharry** [comment]
or maybe you can try Unblock Boundary，like unblock youku but up to date.

**Comment by yan12125** [comment]
Good news - Unblock Boundary works

**Comment by yan12125** [comment]
Thanks for the report (and the handy unblocker :) New Bangumi URLs will be supported in the next version. You can use http://bangumi.bilibili.com/anime/5802/play#100643 to download a single episode or http://bangumi.bilibili.com/anime/5802 to download the whole anime.
